docs/man/socomec_jbus.txt

```diff
@@ -0,0 +1,189 @@
+Socomec_jbus(8)
+==================
+
+NAME
+----
+
+socomec_jbus - Driver for Socomec JBUS UPS with 
+RS-232 serial Modbus connection.
+
+SYNOPSIS
+--------
+
+*socomec_jbus* -h
+
+*socomec_jbus* -a 'DEVICE_NAME' ['OPTIONS']
+
+NOTE: This man page only documents the hardware-specific features of the
+socomec_jbus driver.  For information about the core driver, see
+linkman:nutupsdrv[8].
+
+SUPPORTED HARDWARE
+------------------
+
+This driver supports Socomec JBUS series, online (double conversion)
+UPS with the following characteristics.
+
+1. Single phase and 3-phase UPS
+
+2. Connection: RS-232 
+
+My netvision died out on me and appears impossible to procure another, so
+I set about to talk to the UPS directly. The documentation yielded mild
+success for the DIGYS I have handy over here.
+
+Currently, it has only been tested on the following model.
+
+* DIGYS 3/3 !%kVA
+
+In theory, any Socomec JBUS model should work. It should be discovered
+as ``Unknown Socomec JBUS'' with a numeric id that I'll need to add it
+to the list of supported UPSs. 
+
+Sadly, Socomec document only mentions DELPHYS MX and DELPHYS MX elite and
+I have the id of my own DIGYS, so chances are, your model will not be 
+recognized but will probably mostly work. Please report successful
+or unsuccessful results to the bug tracker or the mailing list.
+Make sure to include the full model number of your UPS manually
+in your report.
+
+socomec_jbus uses the libmodbus project, for Modbus implementation.
+
+CABLING
+-------
+
+The UPS has an RS-232 port which should be connected with a NULL-modem
+cable to a PC serial port. The UPS, mine at least, has curiously a female
+DB9 connector, so if you construct the cable yourself, use a male DB9 on
+one end. 
+
+It only uses RX,TX and ground, so do not strain too much with DTR et all
+
+RS-232 is supported on all operating systems, either via a built-in serial
+port on your computer, or by using an external USB-to-RS-232 converter. If
+you plan to use an USB-to-RS-232 converter, make sure it's supported by your
+operating system.
+
+
+INSTALLATION
+------------
+
+This driver is not built by default.  You can build it by installing libmodbus
+(with development packages) and running
+
+    configure --with-serial=yes --with-modbus=yes
```

Comment:
`./`configure

and actually, I think it should be built "by default" as long as libmodbus-dev (or equivalent) is detected:
````
NUT_ARG_WITH([modbus], [build and install modbus drivers], [auto])
````